src/elements/Divider/Divider.js

```diff
@@ -1,30 +1,81 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import createFragment from 'react-addons-create-fragment'
+
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+/**
+ * A divider visually segments content into groups
+ */
+function Divider(props) {
+  const {
+    horizontal, vertical, inverted, fitted, hidden, section, clearing,
+    children, className,
+  } = props
+
+  const classes = cx('sd-divider ui',
+    useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(vertical, 'vertical'),
+    useKeyOnly(inverted, 'inverted'),
+    useKeyOnly(fitted, 'fitted'),
+    useKeyOnly(hidden, 'hidden'),
+    useKeyOnly(section, 'section'),
+    useKeyOnly(clearing, 'clearing'),
+    'divider',
+    className
+  )
+
+  const _children = createFragment({
+    children,
+  })
+
+  const DividerComponent = 'div'
+  const rest = getUnhandledProps(Divider, props)
+
+  return (
+    <DividerComponent className={classes} {...rest}>
+      {_children}
```

Comment:
Of course.  I'd just whittled down the Label version, and not considered `createFragment` was now redundant